provisioner/powershell/provisioner.go

```diff
@@ -501,6 +508,16 @@ func (p *Provisioner) generateElevatedRunner(command string) (uploadedPath strin
 		log.Printf("Elevated user %s converted to %s after escaping chars special to PowerShell",
 			p.config.ElevatedUser, escapedElevatedUser)
 	}
+	// Replace ElevatedPassword for winrm users who used this feature
+	if strings.Compare(p.config.ElevatedPassword, ".AUTO_WINRM_PASSWORD") == 0 {
```

Comment:
just so I understand correctly: If the user wants to use the auto generated password for the "elevated password", they'll set it to be the literal ".AUTO_WINRM_PASSWORD", and this is needed because sometimes users use an init script to set the password to something else?

Reply:
Yep.